database/gormdb_assignment.go

```diff
@@ -41,6 +41,7 @@ func (db *GormDB) CreateAssignment(assignment *pb.Assignment) error {
 			"is_group_lab":      assignment.IsGroupLab,
 			"reviewers":         assignment.Reviewers,
 			"container_timeout": assignment.ContainerTimeout,
+			"skip_tests":        assignment.SkipTests,
```

Comment:
I've been confused about the naming of this field since I first saw it. Doesn't this just mean `manual`? I mean as in manually reviewed assignment. I'm okay to leave this as-is for now in this PR, but I think we should change the name. As it is now, it sounds as if there are tests for this assignment, but they are to be skipped.

Reply:
Honestly, both `skip_tests` and `manual` sound equally confusing to me out of context. I can change it to `manual`, but I'd rather do it later to avoid delays caused by necessary changes to yaml files for all related assignments, as there is a lab deadline closing in now for one of those.